### PR TITLE
feat: clean up STRINGS for ddl

### PR DESCRIPTION
**Pull request by jrconlin**
Issue #258

Please remember to consult our [contributing guidelines](https://github.com/mozilla-services/syncstorage-rs/blob/master/CONTRIBUTING.md#sending-pull-requests) before opening your PR.

- [ ] **Title** begins with _type_ (fix, feature, doc, chore, etc) and a short description with no period
- [ ] **Description**  outlines the change
- [ ] **Test cases** included in the change (if appropriate)
- [ ] **Closes** or **Issue** link to associated issue(s)

### CODE REVIEW

File: spanner-2019-10-01.ddl [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+-- - mono_num: a monotonically increasing timestamp or generation number 
+--             padded to 13 digits, provided by the fxa server
+-- - client_state: the first 16 bytes of a SHA256 hash of the user's sync
+--             encryption key.
+
 CREATE TABLE user_collections (
-  fxa_uid STRING(MAX)  NOT NULL,
-  fxa_kid STRING(MAX)  NOT NULL,
+  fxa_uid STRING(36)  NOT NULL,
+  fxa_kid STRING(38)  NOT NULL,
```
Comment (jrconlin) [inline spanner-2019-10-01.ddl]: IIUC, 13 + 1 + (16bytes as base64 = 24 ASCII) == 38

Comment (pjenvey) [reply spanner-2019-10-01.ddl]: it's as hex (so 32 ascii) = STRING(46)

maybe we overallocate slightly on the boundaries? so STRING(40) and STRING(48)?

Comment (fzzzy) [reply spanner-2019-10-01.ddl]: Is alignment going to matter in any way? Should they just all be powers of 2, 64?

**Comment by jrconlin** [comment]
Closing in favor of #278, tied to the correct bug.

**Pull request closed without merging**